runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/process/process.json

```diff
@@ -308,9 +308,9 @@
     "Process.Details.Modeler.Actions.SharePointExcel.GetRange.StartCell.Info": "Adres komórki początkowej (np. A1)",
     "Process.Details.Modeler.Actions.SharePointExcel.GetRange.EndCell": "Komórka końcowa",
     "Process.Details.Modeler.Actions.SharePointExcel.GetRange.EndCell.Info": "Adres komórki końcowej (np. H5)",
-    "Process.Details.Modeler.Actions.SharepointExcel.SetCell.Label": "Dodaj zawartość do komórki",
-    "Process.Details.Modeler.Actions.SharePointExcel.SetCell.Content": "Zawartość komórki",
-    "Process.Details.Modeler.Actions.SharePointExcel.SetCell.Cell": "Adres komórki",
+    "Process.Details.Modeler.Actions.SharepointExcel.SetCell.Label": "Ustaw wartość komórki",
```

Comment:
Suggestion: "Ustaw komórkę" (there is "Pobierz komórkę", "Pobierz komórki", and English "Set cell")